File: mo-system/mo-system-api/src/main/java/ink/ckx/mo/system/api/model/entity/SysDictType.java
package ink.ckx.mo.system.api.model.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import ink.ckx.mo.common.core.base.BaseEntity;
import ink.ckx.mo.common.web.enums.StatusEnum;
import lombok.Data;

@Data
public class SysDictType extends BaseEntity {

    @TableId(type = IdType.ASSIGN_ID)
    private Long id;

    private String code;

    private String name;

    private StatusEnum status;

    private String remark;
}

File: mo-admin/mo-admin-api/src/main/kotlin/ink/ckx/mo/admin/api/model/entity/SysDictType.kt
package ink.ckx.mo.admin.api.model.entity

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableId
import ink.ckx.mo.common.core.base.BaseEntity
import ink.ckx.mo.common.web.enums.StatusEnum

data class SysDictType(
    @TableId(type = IdType.ASSIGN_ID)
    var id: Long? = null,
    var code: String? = null,
    var name: String? = null,
    var status: StatusEnum? = null,
    var remark: String? = null,
) : BaseEntity() 